c/polyjuice.h

```diff
@@ -1484,14 +1521,25 @@ int run_polyjuice() {
     for (int i = 0; i < 32; i++) {
       value_ptr[i] = msg.value.bytes[31 - i];
     }
+
+    uint64_t gas_used = min_gas;
+    // charge gas fee of account creation if to_address is not registered
+    gw_reg_addr_t to_addr = new_reg_addr(g_eoa_transfer_to_address.bytes);
+    uint8_t to_script_hash[GW_KEY_BYTES] = {0};
+    ret = context.sys_get_script_hash_by_registry_address(&context, &to_addr, to_script_hash);
+    if (ret == GW_ERROR_NOT_FOUND) {
+        gas_used += NEW_ACCOUNT_GAS;
```

Comment:
The `*gasUsed` should be passed into the `handle_native_token_transfer` function and increased immediately when a new account is created.

Reply:
Done!